Gruntfile.js

```diff
@@ -183,12 +186,10 @@ module.exports = function(grunt) {
         }
 
         eg = file.read(filepath);
-        md = "docs/" + example.file.replace(".js", ".md");
-        inMarkdown = false;
+        name = (example.name || example.file).replace(".js", "");
 
-        if (!example.title) {
-          grunt.fail.fatal("No entry for " + example.file + " in titles.json");
-        }
```

Comment:
The error message about titles.json is obsolete, but it's probably still handy to fail if they forget the required title property? Just change the error message instead?

``` js
grunt.fail.fatal("Invalid example (" + name + "): title required");
```